client/container.go

```diff
@@ -37,6 +39,12 @@ func (x *PrmContainerPut) SetContainer(cnr container.Container) {
 	x.cnrSet = true
 }
 
+// SetSigner sets signer to sign request payload.
```

Comment:
i'd also emphasize that
```
Signer's scheme should be neofscrypto.ECDSA_DETERMINISTIC_SHA256. For example, you can use neofsecdsa.SignerRFC6979.
```

also clarify optionality:
```
Optional: defaults to internal Client signer.  
```

same for delete/seteacl